Ukoly/DU_5/Rodne_cislo.py

```diff
@@ -0,0 +1,56 @@
+birth_number = input('Enter the birth number: ')
+while len(birth_number) != 11:
+    if len(birth_number) != 11:
+        print('The number has to be 11 characters long')
+        birth_number = input('Enter the birth number: ')
+
+format = birth_number.split('/')
+without_slash = ''.join(format)
+
+month_30 = [4, 6, 9, 11]
+month_31 = [1, 3, 5, 7, 8, 10, 12]
+
+if not without_slash.isnumeric:
+    print('You can use only digits or slash "/"')
+if birth_number.count('/') > 1:
+    print('There should be only one separator')
+else:
+    day = int(birth_number[4:6])
+    month = int(birth_number[2:4])
+    if birth_number[6] != '/':
```

Comment:
Toto by bolo možno lepšie overiť skorej, nakoľko ak aktuálne zadáš niečo ako `00041/41234`, tak to umrie na výnimke 😢, nakoľko si sa to o riadok vyššie už pokúsila preparsovať na `int`.